scripts/drupal-update.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+# To setup a dev build you must run:
+# $ export BUILD_ENV=dev
+# prior to starting docker container.
+
+# Get Enviroment Variable.
+build_env=$BUILD_ENV
+
+# Set drush command for container.
+drush="docker-compose run --rm php drush"
+
+# Set build type.
+dev_setup=false
+if [[ $build_env = "dev" ]]; then
+  dev_setup=true
+fi
+
+# For testing.
+echo "BUILD_ENV: $build_env"
+echo "DEV_SETUP: $dev_setup"
+
+# Start build.
+if $dev_setup ; then
+  # Dev build.
+  echo "Building Local Development Enviroment."
+  echo ""
+
+  # Check/Add Drupal settings file.
+  if [ ! -e "sites/default/settings.local.php" ]
+  then
+    echo "Copying over default datbase settings."
+    cp -v "../scripts/settings.local.dist.php" "sites/default/settings.local.php"
+    echo ""
+  fi
+
+  # Install production backup db.
+  if [ -e "../production-db.sql" ] ; then
+
+    # Drop current tables.
+    echo "Dropping all tables."
+    $drush sql-drop -y
+    echo ""
+
+    echo "Importing production backup database."
+    if $drush sqlc < ../production-db.sql ; then
+      echo "Database import successful."
+      echo ""
+    else
+      echo "Database import failed."
+      exit
+    fi
+  else
+    echo "Can not find production-db.sql in project root."
+    echo "Place a production backup named production-db.sql in project root."
+    exit
+  fi
+
+  echo "Changeing Temporary Directory"
+  $drush vset file_temporary_path /tmp
```

Comment:
This doesn't work on D8 :( 

Error: 
Changeing Temporary Directory
Command variable-set requires Drupal core version 6 or 7 to run.                          [error]
The drush command 'vset file_temporary_path /tmp' could not be executed.                  [error]